tests/data.py

```diff
@@ -652,6 +652,18 @@ class Galadriel:
     }
 
 
+parametrized_experiment_params: list[list[str | Circuit | list[Circuit]]] = []
+for platform in [Galadriel]:
+    circuit = Circuit(1)
+    circuit.add(I(0))
+    circuit.add(Align(0))  # Parametrized gate
```

Comment:
You are not passing the parameter though, no?

That `0` is the `qubit`, but the parameter is the delay, which you don't pass, no?

![Screenshot 2024-06-10 at 21 19 17](https://github.com/qilimanjaro-tech/qililab/assets/109400222/96aa58f9-65fe-4e4f-97d7-9def25e66133)

Maybe pass it also, from the start, so its more clear how it is parametrized :)
```suggestion
    circuit.add(Align(0, delay=0.1))  # Parametrized gate
```

Reply:
I was passing the parameter 0 but 'elegantly' 🎩 
Sure I can specify it so its more readable 😄

Reply:
Yes please do that! 
(I think its not been changed in remote yet, with these last commits though, so I'm unresolving this one 👍 )

Reply:
You are right, was not commited hehe (resolving it)